Jenkinsfile

```diff
@@ -12,6 +12,7 @@ node {
     },
     rubyLintDiff: false,
     rubyLintRails: true,
+    rubyLintDirs: "*",
     overrideTestTask: {
```

Comment:
Sorry I know not introduced here, but same world. Can this be switched to be `testTask: "spec"` ?

Reply:
Oh yeah - I wasn't sure how to do that when I wrote it. Done.

Reply:
I actually thought we might have had a testTask option so thought we could get it much more succinct but I was wrong - so apologies if this seemed a bit more minor than it was.